src/main/kotlin/no/nav/k9/domene/repository/StatistikkRepository.kt

```diff
@@ -375,57 +375,27 @@ class StatistikkRepository(
         }
     }
 
-    fun hentReservasjoner(): List<Reservasjon> {
-        val reservasjoner = using(sessionOf(dataSource)) {
+    fun hentTiderFraOpprettetBehandlingTilFørsteReservasjon(): List<Pair<LocalDateTime, LocalDateTime>> {
+        val tiderFraOpprettetBehandlingTilFørsteReservasjon = using(sessionOf(dataSource)) {
             it.run(
-                queryOf(
-                    "select id, opprettet from reservasjon where opprettet is not null"
-                ).map { rad ->
-                    Reservasjon(
-                        reservertTil = null,
-                        reservertAv = "",
-                        flyttetAv = "",
-                        flyttetTidspunkt = null,
-                        begrunnelse = null,
-                        oppgave = UUID.fromString(rad.string("id")),
-                        opprettet = rad.localDateTimeOrNull("opprettet")
+                queryOf("""
+                    select b.opprettet behandling_opprettet, r.opprettet reservasjon_opprettet
```

Comment:
Reservasjonene for en gitt oppgave ligger i en liste i jsondata, som utvides for hver reservasjon. 
Det blir derfor ikke riktig å ha reservasjon_opprettet som et felt, slik strukturen er nå.

Reply:
Ah. Er ikke så lett å teste det lokalt 😅 Da blir vel det nye feltet mer et timestamp på når det ble opprettet en reservasjon på hver oppgave for første gang, men det blir kanskje ikke helt riktig det heller?

Reply:
må kanskje lagre det som json felt da som alt annet for å følge strukturen...?